portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -17,10 +17,10 @@
 /** An item on a todo list. */
 public final class Comment {
   
-  private final long id;
-  private final String username;
-  private final String message;
-  private final long timestamp;
+  public final long id;
```

Comment:
As a general rule, you could consider java accessor methods to avoid exposing the inner working of your class. This can be handled in a follow up if you think so.